src/main/scala/com/fulcrumgenomics/umi/FilterConsensusReads.scala

```diff
@@ -72,6 +76,9 @@ import htsjdk.samtools.reference.ReferenceSequenceFileWalker
       |FilterConsensusReads ... --min-reads 10 5 3 --max-base-error-rate 0.1
       |```
       |
+      |Furthermore, the `--require-single-strand-agreement` option (for duplex consensus reads) will only be applied
```

Comment:
Should this be in the numbered list of filters?  Something like:

```
7. For duplex reads, if --require-single-strand-agreement is provided, masks to N any bases where the base was observed in both single-strand consensus reads and the two reads did not agree.
```

Reply:
Just to build on this ... from the usage and arg documentation alone it's not clear to me how this filter works (it is in code).  I.e. I could interpret this as masking bases, or as requiring this be true of all bases in the read otherwise reject the read.